schemas/2014-04-01/Microsoft.Sql.json

```diff
@@ -84,6 +87,55 @@
       ],
       "description": "Microsoft.Sql/servers"
     },
+    "servers_administrators": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "oneOf": [
+            {
+              "type": "string",
+              "enum": [
+                "activeDirectory"
```

Comment:
>activeDirectory [](start = 17, length = 15)

Is this correct? I don't think ARM accepts this name. For nested resources, the name needs to be something like "serverName/adminName".